sample/src/main/java/com/airbnb/airmapview/sample/MainActivity.java

```diff
@@ -94,16 +94,17 @@
   @Override public void onMapInitialized() {
     appendLog("Map onMapInitialized triggered");
     final LatLng airbnbLatLng = new LatLng(37.771883, -122.405224);
-    addMarker("Airbnb HQ", airbnbLatLng);
-    addMarker("Performance Bikes", new LatLng(37.773975,-122.40205));
-    addMarker("REI", new LatLng(37.772127, -122.404411));
+    addMarker("Airbnb HQ", "snippet", airbnbLatLng);
+    addMarker("Performance Bikes", "snippet", new LatLng(37.773975, -122.40205));
+    addMarker("REI", "snippet", new LatLng(37.772127, -122.404411));
     map.animateCenterZoom(airbnbLatLng, 10);
   }
 
-  private void addMarker(String title, LatLng latLng) {
+  private void addMarker(String title, String snippet, LatLng latLng) {
```

Comment:
for the sample, can you overload `addMarker` to keep the 2 argument method, and only use the method to include the snippet if it is actually used? we're setting the snippet here but aren't using it for anything - it's the same value for each marker